modules/distribution/product/src/main/assembly/bin.xml

```diff
@@ -56,6 +56,9 @@
                 need to remove duplicate classes with jaxb-* bundle -->
                 <exclude>**/repository/components/plugins/wadl-core_1.1.3.wso2v2.jar</exclude>
                 <exclude>**/conf/tomcat/carbon/WEB-INF/web.xml</exclude>
+
+                <!-- Remove multiple versions of same jar-->
+                <exclude>**/repository/components/plugins/httpclient_4.3.2.wso2v1.jar</exclude>
```

Comment:
This fix is a hack. Need to figure out what is the feature that provides old version of  httpclient and get it update to latest version of httpclient. Anyway I will merge this as a temporary fix. But make sure to remove this from bin.xml and update the feature as mentioned.